pennylane/qnode.py

```diff
@@ -331,8 +331,9 @@ def construct(self, args, kwargs=None):
 
             res = tuple(res)
         else:
-            raise QuantumFunctionError("A quantum function must return either a single measured observable "
-                                       "or a nonempty sequence of measured observables.")
+            raise QuantumFunctionError(
+                "A quantum function must return either a single measured observable "
+                "or a nonempty sequence of measured observables.")
```

Comment:
I've been going back and forth over whether commenting on these white-space changes are nitpicking, but I'm thinking they should be reverted, simply to avoid unnecessary merge conflicts with other open PRs once merged.

Note that once #302 has been done, this issue should not occur any more, as all PRs will have consistent whitespace changes.